src/deb/rhino.sysv

```diff
@@ -0,0 +1,303 @@
+#!/bin/sh
```

Comment:
Please explain which parts of this file are boilerplate and which ones are Rhino-specific